functions/Get-DbaAgentJob.ps1

```diff
@@ -4,11 +4,12 @@ FUNCTION Get-DbaAgentJob {
 			Gets SQL Agent Job information for each instance(s) of SQL Server.
 
 		.DESCRIPTION
-			The Get-DbaAgentJob returns connected SMO object for SQL Agent Job information for each instance(s) of SQL Server.
+			The Get-DbaAgentJob returns connected SMO object for SQL Agent Job information for each instance(s) of SQL Server. The default behavior
+			is to return enabled jobs only.
```

Comment:
SO you are saying we should allow a parameter to accept the string version of "true" and "false"?